packages/oui-button/src/button.component.js

```diff
@@ -0,0 +1,17 @@
+import template from './button.html'
+import controller from './button.controller'
+
+export default {
+  bindings: {
+    text: '@',
+    id: '@?',
+    name: '@?',
+    type: '@?',
```

Comment:
Let me know if it's excessive:
If we create a binding for this attribute, you let the user thinking that the notation `type="{{type}}"` could make it dynamic. However this is not the case here: `type` will never be re-parsed internally if it changes (we assume that's right, the type of a button is almost never updated).
1) We could just rely on $attrs instead of bindings if the value is not dynamic.
2) The documentation should be more clear IMO if you omit "@?" (this point is valid only if the (1) is valid).

Reply:
If I understand, you would remove `type: '@?'` and add 'this.type = this.$attrs['type']` on $onInit ?

IMO, I think we should keep it simple.
It's clearer for the dev (listed in bindings) and used in an Angular way.

Reply:
I think @neolitec is right, we should not let the user think that attribut is a real binding when we don't support when value is updated. Anyway, `type` is not the only attribute that should be documented outside bindings, `primary|secondary|link` for example. Also, we are giving a `README.md` file fully documented about that, I think we can do it.

Reply:
Seen together, we'll add it on the doc that it's a one-time binding :)